lib/rpc/server/agent.go

```diff
@@ -156,15 +156,6 @@ func (srv *agentServer) Abort(ctx context.Context, req *types.Empty) (resp *type
 }
 
 func (srv *agentServer) command(req pb.CommandArgs, stream pb.Agent_CommandServer, log *log.Entry) (err error) {
-	defer func() {
```

Comment:
When the bug above triggered panic, it was impossible to see where it failed. In general, I don't think it's useful to consume the panics as here it is a definite programming error.